src/index.js

```diff
@@ -260,6 +274,94 @@ const createElement = (tagType, attrObj, parentElement) => {
 
     return newElement
 }
+
+const createGlobalHeader = ({ headerTitle, button1Label, button1OnClick, button2Label, button2OnClick, parentElement }) => {
+    const headerElement = createElement("header", undefined, parentElement)
+    createElement("button", {
+        innerText: button1Label,
+        // TODO: make it more generic
+        className: CLASS_NAMES.headerButtons,
+        onclick: button1OnClick
+    }, headerElement)
+
+    createElement("p", {
+        // TODO: make it more generic
+        id: ELEMENT_IDS.contactsTitle,
+        innerText: headerTitle
+    }, headerElement)
+
+    createElement("button", {
+        innerText: button2Label,
+        // TODO: make it more generic
+        className: CLASS_NAMES.headerButtons,
+        onclick: button2OnClick
+    }, headerElement)
+}
+
+const viewContact = () => {
+    handleFooterElementOnClick("view-contact")
+    removeElementsOnTabChange("view-contact")
+    createGlobalHeader({
+        headerTitle: "",
+        button1Label: INNER_TEXTS.backButtonElement,
+        button1OnClick: renderContacts,
+        button2Label: INNER_TEXTS.editButtonElement,
+        button2OnClick: console.log("This will edit the contact"),
```

Comment:
Are you sure this is working (even if it is console.log). My guess is that this console.log is fired even without clicking on button2 (which is incorrect)